WePool/src/app/location-autocomplete/location-autocomplete.component.html

```diff
@@ -0,0 +1,13 @@
+<!Container with Angular Material form fields to take input>
+<div id="container">
+    <mat-form-field id="origin">
+        <mat-label>Origin</mat-label>
+        <input matInput id="originInput" value="">
+    </mat-form-field>
+    <p></p>
+    <mat-form-field id="destination">
+        <mat-label>Destination</mat-label>
+        <input matInput id="destinationInput" value="">
+    </mat-form-field>
+    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyB7Y7JJMsGZP_DabHWCv2A2iLwuoP5UcCs&libraries=places"></script>
```

Comment:
API key is hard-coded in URL. Instead, it should come from global constants.

Reply:
It seems we can't avoid having the API key public for Google Maps. We can restrict the API key instead. Ignore this comment.